app/assets/stylesheets/mtl/extend/_chips.scss

```diff
@@ -1,3 +1,22 @@
 .chip {
+  border-radius: $chip-border-radius;
+  height: $chip-height;
+  line-height: $chip-line-height;
+  font-size: $chip-font-size;
+  font-weight: $chip-font-weight;
   white-space: nowrap;
+  padding-right: $chip-height / 2;
+
+  > img {
+    border-radius: $chip-border-radius;
+    height: $chip-height;
+    margin-right: $chip-height / 4;
+    width: $chip-height;
+  }
+
+  .close {
+    font-size: $chip-font-size + 3px;
+    height: $chip-height;
+    line-height: $chip-line-height;
```

Comment:
here, shouldn't we add a padding-left as 1/4 of the font-size to keep the proper alignment, like we did in the img margin right? Asking.